Makefile

```diff
@@ -85,16 +85,15 @@ generate-samples: install
 	go run ./hack/samples/generate_samples.go
 
 .PHONY: generate-testdata ## Generate the full testdata directory
-generate-testdata: generate-samples
+generate-testdata:
+	docker login https://registry.redhat.io
+	make generate-samples
```

Comment:
Was this not working as a prerequisite? Is there something different/better about calling it explicitly? I think calling it this way will launch a 2nd make as a sub-task to run it.

Reply:
because I add the docker login there to make it easier for others to understand and to force that is a pre-requirement.

Reply:
Those seem like reasonable justifications to doing it this way. 👍